### PR TITLE
[stable/mongodb] fix README.md according to values.yaml

### PR DESCRIPTION
**Pull request by LittleWhiteYA**
#### What this PR does / why we need it:

- fixme README.md values according to values.yaml

#### Checklist
[Place an '[x]' (no spaces) in all applicable fields. Please remove unrelated fields.]
- [x] [DCO](https://github.com/helm/charts/blob/master/CONTRIBUTING.md#sign-your-work) signed
- [x] Chart Version bumped
- [x] Variables are documented in the README.md
- [x] Title of the PR starts with chart name (e.g. `[stable/chart]`)

### CODE REVIEW

File: stable/mongodb/values.yaml [hunk 1]
```diff
@@ -313,7 +313,10 @@ metrics:
   ## Metrics exporter resource requests and limits
   ## ref: http://kubernetes.io/docs/user-guide/compute-resources/
   ##
-  # resources: {}
+  resources:
```
Comment (sameersbn) [inline stable/mongodb/values.yaml]: You should revert this change. We do not suggest adding `resource.requests` in the default configuration.

**Review by sameersbn — CHANGES_REQUESTED** [review]

File: stable/mongodb/values.yaml (hunk 1 above)
Comment (LittleWhiteYA) [reply stable/mongodb/values.yaml]: sure. I remove README.md default values

**Review by sameersbn — APPROVED** [review]

File: stable/mongodb/values.yaml (hunk 1 above)
Comment (sameersbn) [reply stable/mongodb/values.yaml]: perfect, Thanks!
